src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/ExceptionRecordMapper.java

```diff
@@ -105,35 +107,54 @@ private String extractSurnameFromOcrData(Envelope envelope) {
             return null;
         }
 
-        String surnameOcrFieldName = serviceConfigProvider.getConfig(envelope.container)
-            .getSurnameOcrFieldName(envelope.formType);
 
-        List<String> surnameList = envelope.ocrData
-            .stream()
-            .filter(ocrData -> ocrData.name.equals(surnameOcrFieldName))
-            .map(ocrData -> ocrData.value)
-            .collect(toList());
-        if (surnameList.size() == 0) {
-            LOGGER.info(
-                "Surname not found in OCR data. "
-                    + "Surname Ocr Field Name: {}. Envelope id: {}, Case Ref: {}, Jurisdiction: {}",
-                surnameOcrFieldName,
-                envelope.id,
-                envelope.caseRef,
-                envelope.jurisdiction
-            );
+        Optional<List<String>> surnameOcrFieldNameListOp =
+            serviceConfigProvider
+                .getConfig(envelope.container)
+                .getSurnameOcrFieldNameList(envelope.formType);
+
+
+        if (!surnameOcrFieldNameListOp.isPresent()) {
             return null;
-        } else if (surnameList.size() > 1) {
-            LOGGER.error(
-                "Surname found {} times in OCR data. "
-                    + "Surname Ocr Field Name: {}, Envelope id: {}, Case Ref: {}, Jurisdiction: {}",
-                surnameList.size(),
-                surnameOcrFieldName,
-                envelope.id,
-                envelope.caseRef,
-                envelope.jurisdiction
-            );
         }
-        return surnameList.get(0);
+
+
+        for (String surnameOcrFieldName : surnameOcrFieldNameListOp.get()) {
+
+            LOGGER.info("Surname search by ocr field name {}", surnameOcrFieldName);
+
+            List<String> surnameList = envelope.ocrData
+                .stream()
+                .filter(ocrData -> ocrData.name.equals(surnameOcrFieldName) && StringUtils.isNotBlank(ocrData.value))
+                .map(ocrData -> ocrData.value)
+                .collect(toList());
+
+            if (!surnameList.isEmpty()) {
+
+                if (surnameList.size() > 1) {
```

Comment:
do we need this condition? We can always get the first one when list is not empty ?

Reply:
we want to log?

Reply:
I personally never wanted this log. But it is not my call right now

Reply:
On other note, if we agree on skipping the log, `findFirst` before map would be useful